open-craft/cert-manager
# SE-2197: pin certbot version

Pin certbot version to match the versions deploy across all systems.

* The version of `certbot` and `acme` is downgraded to `0.31.0` (these should be in lock-step)
* CircleCI is configured to use the pinned versions
* A simple test has been added to ensure the pinning actually works

**Jira tickets:** SE-2197

**Testing instructions:**

From a checkout of the cert-manager project:
```
pip install --user pipenv
pipenv install --dev --ignore-pipfile
pipenv pytest cert_manager/test_versions.py
pipenv pytest cert_manager/test_cert_manger.py
pipenv pytest cert_manager/test_cert_utils.py
```
Note the backend tests will only work if you have consul set up locally (see SE-2442).

**Reviewers**
